CHANGELOG.md

```diff
@@ -0,0 +1,15 @@
+# Changelog
+
+## 1.0.0 (April 24, 2017)
+
+### Features:
+
+* Use `cardano-sl` version `0.4.0`
```

Comment:
Well, `cardano-sl-0.4.3` has fixes which are relevant only for nodes which create blocks. Explorer is not such node. However, I still recommend you to use 0.4.3, because it also contains at least some logging improvements which may be valuable if something goes wrong.

> Also, its an imperative that we use the same version of cardano-sl that is deployed in production

Yes, it makes sense.